drivers/tripplite_usb.c

```diff
@@ -576,7 +576,7 @@ static int send_cmd(const unsigned char *msg, size_t msg_len, unsigned char *rep
 		}
 
 #if ! defined(__FreeBSD__)
-		if(!done) { usleep(1000*100); /* TODO: nanosleep */ }
+		usleep(1000*100); /* TODO: nanosleep */
```

Comment:
Why this change? Originally we slept if not "done", now we'd delay for sleep even if "done"... any physical reason for that?

- [ ] Not sure about dropped condition check

Reply:
`!done` is always true here because it's a condition of the `for` loop, at line 567.

identicalInnerCondition,drivers/tripplite_usb.c:579,warning,Identical inner 'if' condition is always true.